pixie/stdlib.pxi

```diff
@@ -1565,6 +1565,20 @@ For more information, see http://clojure.org/special_forms#binding-forms"}
         val
         nil))))
 
+;; TODO: place withing Range type when protocols support multi-arity functions
```

Comment:
A nitpick: `s/withing/within/`.

Also, you could place it within Range and use `range->seq` as an inner function. Especially because `-seq` has arity 1.